src/main/java/com/mraof/minestuck/entity/consort/ConsortEntity.java

```diff
@@ -402,4 +407,55 @@ public static boolean canConsortSpawnOn(EntityType<ConsortEntity> entityType, IW
 	{
 		return true;
 	}
+
+	@Override
+	public void registerControllers(AnimationData data) {
+		data.addAnimationController(createAnimation("walkAnimation", 1, this::walkAnimation));
+		data.addAnimationController(createAnimation("armsAnimation", 1, this::armsAnimation));
+		data.addAnimationController(createAnimation("deathAnimation", 1, this::deathAnimation));
+		data.addAnimationController(createAnimation("actionAnimation", 1, this::actionAnimation));
+	}
+
+	private <E extends IAnimatable> PlayState walkAnimation(AnimationEvent<E> event) {
+		if (!event.isMoving()) {
+			return PlayState.STOP;
+		}
+
+		event.getController().setAnimation(new AnimationBuilder().addAnimation("walk", true));
+		return PlayState.CONTINUE;
+	}
+
+	private <E extends IAnimatable> PlayState armsAnimation(AnimationEvent<E> event) {
+		if (!event.isMoving()) {
+			if (this.getConsortType() == EnumConsort.TURTLE) { // eeeeeeeeeeehhh maybe just fix the turtle anims instead
+				event.getController().setAnimation(new AnimationBuilder().addAnimation("armfix", true));
+				return PlayState.CONTINUE;
```

Comment:
Is this something you want me to ask of Riotmode?

Reply:
😁 I already discussed that with em, I forgot I had to run a script on the model